.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
Can consider "unstaging" these configuration file as it should not be part of the code base :-).